Classes/Controller/Application/SubmitController.php

```diff
@@ -37,12 +43,60 @@ public function showSummaryAction(Application $application)
      */
     public function submitAction(Application $application)
     {
-        
+
         $application->submit();
 
         $this->repository->updateAndLog(
             $application,
             'new'
         );
+
+        if (ExtconfService::getInstance()->getSendAutoAcknowledge()) {
+            $this->forward("sendAutoAcknowledgement", null, null, ['application' => $application]);
+        } else {
+            $this->redirect("submited", null, null, ['application' => $application]);
+        }
+    }
+
+    /**
+     * @param  Application $application
+     * @return void
+     */
+    public function submitedAction(Application $application)
```

Comment:
Typo in action name, "submited" should be "submitted".

Reply:
Why are the two additional methods necessary? Couldn't this also be done in the SubmitAction (while moving the mail stuff into a service function to slim down controller code) and/or in a separate, non-action function in the controller?

Reply:
If everything is done in the submit action the email/application would be resend on page reload.
I am going to move the e-mail stuff into the submit action(Service/Function) as it makes sense, but keep the submitted action.